packages/node-core/src/indexer/testing.service.ts

```diff
@@ -204,6 +208,17 @@ export abstract class TestingService<A, SA, B, DS> {
         }
       }
 
+      //remove tested entities from DB
+
+      await Promise.all(
+        entitiesCreated.map(async (entity) => {
+          // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+          await store.remove(entity._name!, entity.id);
+        })
+      );
```

Comment:
Why is this necessary? Dropping the schema should be enough

Reply:
dropping the schema and recreating it immediately seems to retain the records from before drop. I'm unable to find any docs related to this but the references are still intact.

Reply:
We should try and figure out a solution rather than a work around